src/Ocelot/DependencyInjection/ConfigurationBuilderExtensions.cs

```diff
@@ -39,19 +40,17 @@ public static IConfigurationBuilder AddOcelot(this IConfigurationBuilder builder
 
         public static IConfigurationBuilder AddOcelot(this IConfigurationBuilder builder, string folder, IWebHostEnvironment env)
         {
+            Regex.CacheSize += 100;
```

Comment:
Do you consider the `AddOcelot` methos as an entry point for entire Ocelot pipeline, and for all regex-objects of the app domain?

Reply:
`100` hardcoded value should be removed in favor of a static property!

Fixed in https://github.com/ThreeMammals/Ocelot/pull/1348/commits/364d1b63ee39370f53c5e96bce2395d92b068be5 ✔️